diff -urpN libmspack-0.5alpha-orig/test/chmx.c libmspack-0.5alpha/test/chmx.c
--- libmspack-0.5alpha-orig/test/chmx.c	2016-08-23 11:32:28.000000000 -0300
+++ libmspack-0.5alpha/test/chmx.c	2016-08-23 11:34:17.000000000 -0300
@@ -32,7 +32,7 @@ static int ensure_filepath(char *path) {
     if (*p != '/') continue;
     *p = '\0';
     ok = (stat(path, &st_buf) == 0) && S_ISDIR(st_buf.st_mode);
-    if (!ok) ok = (mkdir(path, 0777 & ~user_umask) == 0);
+    if (!ok) ok = (mkdir(path) == 0);
     *p = '/';
     if (!ok) return 0;
   }
