--- ../backports.lzma-backports.lzma.v0.0.6/setup.py	2016-06-06 00:42:13.000000000 -0700
+++ setup.py	2016-06-30 20:37:18.000000000 -0700
@@ -13,6 +13,12 @@
 from distutils.core import setup
 from distutils.extension import Extension
 
+from functools import partial
+
+def get_export_symbols(*a, **kw):
+    return []
+build_ext.get_export_symbols = get_export_symbols
+
 # We now extract the version number in backports/lzma/__init__.py
 # We can't use "from backports import lzma" then "lzma.__version__"
 # as that would tell us the version already installed (if any).
